app/statefull_flow/objective.yaml

```diff
@@ -39,7 +39,7 @@ components:
           x-field-uid: 1
         ramp_up_type:
           description: >-
-            Ramp up type for test.
+            Ramp up type to either bring up users at a certain rate.
```

Comment:
The description is still not fully clear to me.

Reply:
Updated in PR